docs/api-and-integration/octopus.server.exe-command-line/service.md

```diff
@@ -8,28 +8,30 @@ Use this to configure and manage your Octopus service.
 **Service options**
 
 ```text
-Usage: Octopus.Server service --start [<options>]
+Usage: octopus.server service [<options>]
 
 Where [<options>] is any of:
-      --instance=VALUE               Name of the instance to start
 
-Usage: Octopus.Server service --stop [<options>]
```

Comment:
This old separation was nicer, but I suspect the win of automating the updates will trump this